lib/shared/permissions.js

```diff
@@ -143,14 +158,18 @@ exports.getEnvironmentCommandPrefix = (environment) => {
  * });
  */
 exports.elevateCommand = (command, options) => {
-  const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
+  const prefixedCommand = _.concat(
+    exports.getEnvironmentCommandPrefix(options.environment),
+    _.map(command, quoteString)
```

Comment:
I'll have to test, but I think this might cause problems on Linux in the (unlikely) scenario that a path contains a `"` ?